test/util/malicious/out_of_order_prepare.go

```diff
@@ -18,19 +19,14 @@ import (
 // for. It will swap the order of two blobs in the square and then use the
 // modified nmt to create a commitment over the modified square.
 func (a *App) OutOfOrderPrepareProposal(req *abci.RequestPrepareProposal) (*abci.ResponsePrepareProposal, error) {
-	ctx := a.NewContext(false)
```

Comment:
this blew up with a nil pointer, but no need for the ctx as we can just pass latest version I believe